CHANGELOG.unreleased.md

```diff
@@ -30,5 +30,6 @@
 <!--packages-start-->
 
 - xo-server-netbox minor
+- xo-web patch
```

Comment:
Don't you want to add a CHANGELOG entry for this?

Reply:
Yes, I'll add an entry!